Dockerfile

```diff
@@ -36,6 +35,24 @@ ARG OFFLINE_SEMGREP_RULESETS="p/ci"
 ENV OFFLINE="${OFFLINE}"
 ENV OFFLINE_AT_BUILD_TIME="${OFFLINE}"
 ENV OFFLINE_SEMGREP_RULESETS="${OFFLINE_SEMGREP_RULESETS}"
+#
+# Allow for UID and GID to be set as build arguments to this Dockerfile.
+#
+# Set the default values of UID/GID to non-zero (and above typical low-number UID/GID values which
+# are defined by default).  500 and 100 are not special values or depended upon.
+# Their selection as default values is only to set the default for this container to be non-zero.
+#
+# The actions performed in this image (CMD command) do not require root privileges.
+# Thus, UID/GID are set non-zero to avoid someone inadvertantly running this image in a container
+# that runs privileged and as as root(UID 0).
+#
+# If the environment in which the container will run requires a specific UID/GID, then --build-arg
+# arguments on the ${OCI_RUNNER} build command can be used to over-ride these values.
+#
+ARG UID=500
+ARG GID=100
+ARG ASHUSER_HOME=/home/ash-user
```

Comment:
This should make `ash-user` a variable, then use it here and below in the `adduser` command.

Reply:
Added variables for both user and group.